test_howdoi.py

```diff
@@ -106,7 +110,57 @@ def test_answers_duckduckgo(self):
             self.assertValidResponse(self.call_howdoi(query))
 
         os.environ['HOWDOI_SEARCH_ENGINE'] = ''
-
+    
+    def create_plugin_file(self, filename, content):
+        path = os.path.join(howdoi.PLUGIN_DIR, filename)
+        with open(path, 'w+') as f:
+            json.dump(content, f)
```

Comment:
Can we expose this to the user to let them generate plugins after just inspecting an element?

Reply:
What do you mean with this?